build.gradle

```diff
@@ -0,0 +1,117 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java project to get you started.
+ * For more details take a look at the Java Quickstart chapter in the Gradle
+ * User Manual available at https://docs.gradle.org/6.6.1/userguide/tutorial_java_projects.html
+ */
+
+plugins {
+    id 'java'
+    id "io.qameta.allure" version "2.8.1"
+}
+
+apply plugin: 'idea'
+apply plugin: 'checkstyle'
+apply plugin: 'pmd'
+apply plugin: 'jacoco'
+apply plugin: "io.qameta.allure"
+
+group 'org.fundacionjala'
+version '1.0'
+sourceCompatibility = 1.11
+
+repositories {
+    mavenCentral()
+}
+dependencies {
+    compile group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.14.0'
+    compile group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.14.0'
+    implementation group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.30'
+    implementation group: 'io.rest-assured', name: 'rest-assured', version: '4.3.2'
+    implementation group: 'io.rest-assured', name: 'json-schema-validator', version: '4.3.2'
+    testImplementation group: 'io.cucumber', name: 'cucumber-java', version: '6.9.0'
+    testImplementation group: 'io.cucumber', name: 'cucumber-testng', version: '6.9.0'
+    testImplementation group: 'io.cucumber', name: 'cucumber-picocontainer', version: '6.9.0'
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '3.6.0'
+    testImplementation group: 'org.testng', name: 'testng', version: '7.3.0'
+    testImplementation group: 'io.qameta.allure', name: 'allure-cucumber6-jvm', version: '2.13.6'
+}
+
+checkstyle {
+    toolVersion = '8.37'
+    configFile = file("${rootProject.rootDir}/config/checkstyle/checkstyle.xml")
+}
+
+pmd {
+    toolVersion = '6.0.0'
+}
+
+tasks.withType(Pmd) {
+    ruleSets = ["java-basic", "java-braces", "java-unnecessary", "java-unusedcode", "java-strictexception"]
+    reports {
+        xml.enabled = false
+        html.enabled = true
+    }
+}
+
+jacocoTestReport {
+    reports {
+        xml.enabled = true
+        html.enabled = true
+    }
+}
+
+check.dependsOn jacocoTestReport
+
+test {
+    useTestNG()
+    exclude 'org/fundacionjala/fc/runner/**'
```

Comment:
Please consider replacing `org/fundacionjala/fc/runner/**` with `org/fundacionjala/pivotal/runner/**`. Same for the nex line.